README.md

```diff
@@ -540,6 +542,71 @@ We have a small collection of algorithms, one for every day of the month. Scroll
   - **Resources:**
       - [Support File (Exchange Rates.csv)](src/docs/Dec21-Exchange_Rates.csv)
 
+
+## **December 22 - Huffman Encoding**
+  - **Problem**
+    - Huffman coding is a lossless data compression algorithm. The idea is to assign variable-length codes to input characters, lengths of the assigned codes are based on the frequencies of corresponding characters. The most frequent character gets the smallest code and the least frequent character gets the largest code.
```

Comment:
Nooooot the Wikipedia entry :P

Use this:

Huffman coding is a lossless data compression algorithm. This means that it attempts to compress the data in a file or other medium without losing any data at all. This is essential in places where we can't afford to lose any data.

Normally, in a text file each character is stored as eight bits. For example, the ASCII character for 'a' is 01100001 (corresponding to the number 97). In a Huffman-encoded file, we attempt to use fewer bits for characters that occur more frequently. In that way, we can drastically reduce the overall size of a file. If half of a file is the letter 'a', then by reducing it from 8 bits to 1 bit, we save 7 bits per occurrence of 'a'. That's a lot of bits.

So how exactly does it work?

Reply:
Okay actually I just realized that this will be a bit of a pain for newbies to implement. They need to know how trees work and do it. And the code is pretty complicated actually. I think we should get rid of this one .-.

Sorry for the trouble.

Reply:
Will change em!